src/main/java/br/unicamp/cst/core/entities/Mind.java

```diff
@@ -38,8 +40,9 @@ public Mind() {
 		codeRack = new CodeRack();
 
 		rawMemory = new RawMemory();
+                codelets = new ConcurrentHashMap();
 
-		motivationalSubsystemModule = new MotivationalSubsystemModule();
+		//motivationalSubsystemModule = new MotivationalSubsystemModule();
```

Comment:
Same as before - remove commented code from final version of the lib.

Reply:
OK ! I will do that and with all other code comments.